tests/providers/aws/services/backup/backup_service_test.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime
+from unittest.mock import patch
+
+import botocore
+
+from prowler.providers.aws.lib.audit_info.audit_info import current_audit_info
+from prowler.providers.aws.services.backup.backup_service import Backup
+
+# Mock Test Region
+AWS_REGION = "eu-west-1"
+
+# Mocking Access Analyzer Calls
+make_api_call = botocore.client.BaseClient._make_api_call
+
+
+def mock_make_api_call(self, operation_name, kwarg):
+    """
+    Mock every AWS API call using Boto3
+
+    As you can see the operation_name has the list_analyzers snake_case form but
```

Comment:
Delete this two lines please or make it general, since it references another service